Testscripts/Linux/utils.sh

```diff
@@ -2833,11 +2833,11 @@ function collect_VM_properties () {
 	echo ",LIS Version,"$(get_lis_version) >> $output_file
 	echo ",Host Version,"$(get_host_version) >> $output_file
 	echo ",Total CPU cores,"$(nproc) >> $output_file
-	echo ",Total Memory,"$(free -h|grep Mem|awk '{print $2}') >> $output_file
-	echo ",Resource disks size,"$(lsblk|grep "^sdb"| awk '{print $4}')  >> $output_file
-	echo ",Data disks attached,"$(lsblk | grep "^sd" | awk '{print $1}' | sort | grep -v "sd[ab]$" | wc -l)  >> $output_file
+	echo ",Total Memory,"$(free -h | grep Mem | awk '{print $2}') >> $output_file
+	echo ",Resource disks size,"$(lsblk | grep "^sdb" | awk '{print $4}') >> $output_file
+	echo ",Data disks attached,"$(lsblk | grep "^sd" | awk '{print $1}' | sort | grep -v "sd[ab]$" | wc -l) >> $output_file
 	echo ",eth0 MTU,"$(cat /sys/class/net/eth0/mtu) >> $output_file
-	echo ",eth1 MTU,"$(cat /sys/class/net/eth1/mtu) >> $output_file
+	[ -f /sys/class/net/eth1/mtu ] && echo ",eth1 MTU,"$(cat /sys/class/net/eth1/mtu) >> $output_file
```

Comment:
I assume this is VF interface. Right? Or, is it the multi synthetic NIC cases? If this falls into the first case, I am afraid of its renaming during the booting. That is the expected case like rename3, rename4, etc. Please clarify.

Reply:
It would be good if the all the interfaces are determined dynamically (like ```ip -a```) and then iterate through it to capture their MTU. Other changes looks good.